packages/core/src/state-management/repository.ts

```diff
@@ -152,7 +153,7 @@ export class Repository {
       }
 
       if (!stream) {
-        stream = await this.fromNetwork(streamId, opts);
+        stream = await this.fromNetwork(streamId);
```

Comment:
I think we need to make sure in the `sync` call below that if the sync never learns of any newer tip than the genesis commit that we still go and do the signature verification that we opted out of in `fromNetwork`